view/rooms/app.js

```diff
@@ -35,9 +35,9 @@ let _io;
 let _user;
 let _roomName;
 let _isActive;
-let botInfo = {};
+const botInfo = {};
 const DEBUG_REALTIME = window.location.href.split(/[&\?]/)
-  .includes('debug=REALTIME');
+  .includes('log=REALTIME');
```

Comment:
I liked the consistency of knowing the same query parameter works in room and perspectives... why changing it?

Reply:
In our bdk we implemented a  "Winston" like console logging 
https://git.soma.salesforce.com/Refocus-Bots/refocus-bdk/pull/24

There will be another PR that will introduce the same logging we use for the bots to refocus; this was a start to move them closer to each other.